webapp/src/projects.tsx

```diff
@@ -1178,7 +1178,7 @@ export class NewProjectDialog extends data.Component<ISettingsProps, NewProjectD
         const langOpts: sui.SelectItem[] = [
             {
                 value: pxt.editor.LanguageRestriction.Standard,
-                display: lf("All Languages")
+                display: python ? lf("Blocks, {0}, and {1}", "JavaScript", "Python") : lf("Blocks and {0}", "JavaScript")
```

Comment:
what about "blocks and python"?

Reply:
As in another language restriction option that hides ts? Easy to do if we want

Reply:
Lets keep it to 3 options for now. Too many options will be confusing.